include/deal.II/dofs/dof_tools.h

```diff
@@ -1453,24 +1455,31 @@ namespace DoFTools
    * @ref GlossBoundaryIndicator "Glossary entry on boundary indicators"
    */
   template <int dim, int spacedim>
-  void
+  DEAL_II_DEPRECATED void
```

Comment:
Its not that important (since this is an obscure function) but IIRC we agreed to mark these things as `DEAL_II_DEPRECATED_EARLY`. Would you mind changing this?

Reply:
Alternatively, do you intend on removing this function right after 9.3 is released?

Reply:
I'm agnostic to it. I'll make the change. I made the same change in #11996 as well.